include/diy/master.hpp

```diff
@@ -86,21 +86,27 @@ namespace diy
         size_t  size;
       };
 
+      struct MessageInfo
+      {
+        int from, to;
+        int round;
+      };
+
       struct InFlightSend
       {
         std::shared_ptr<MemoryBuffer> message;
         mpi::request                  request;
 
         // for debug purposes:
-        int from, to;
+        MessageInfo info;
       };
 
       struct InFlightRecv
       {
-        InFlightRecv() : from(-1), to(-1) {}
+        InFlightRecv() { info.from = info.to = -1; }
```

Comment:
Did you mean the new c++ feature of in-class initialization? Like this:
```C++
      struct InFlightRecv
      {
        MemoryBuffer message;
        MessageInfo info = { -1, -1, -1 };
      };
```
Is this preferred over constructor initialization?